azure-pipelines.yml

```diff
@@ -1,14 +1,49 @@
 # CI and PR triggers
 trigger:
-- main
-- dev16.1
-- feature/*
-- release/*
+  branches:
+    include:
+    - main
+    - dev16.1
+    - feature/*
+    - release/*
+  paths:
+    include:
+    - '*'
+    exclude:
+    - .github/*
+    - docs/compiler-guide.md
```

Comment:
Maybe we should exclude the whole docs directory?

Reply:
I'm hesitant to exclude these since there are `.fsx` files.  They're currently not used during CI, but if that ever changes, I don't want to miss them.